src/main/java/emissary/output/DropOffPlace.java

```diff
@@ -18,19 +22,26 @@
 import emissary.output.filter.IDropOffFilter;
 import emissary.place.ServiceProviderPlace;
 import emissary.util.DataUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * DropOffPlace manages the output from the system It has evolved into a controller of sorts with way too many options,
  * that controls which types of output are desired and called the appropriate output helper for the desired output.
  **/
 public class DropOffPlace extends ServiceProviderPlace implements emissary.place.EmptyFormPlace {
 
+    private static Logger objectMetricsLog = null;
     protected boolean doSynchronized = false;
     protected Set<String> elideContentForms;
     protected Set<String> noNukeForms;
     protected List<IDropOffFilter> outputFilters = new ArrayList<>();
     protected boolean failurePolicyTerminate = true;
     protected DropOffUtil dropOffUtil;
+    private boolean outputObjectMetrics = false;
+    private List<String> objectMetricsFields = new ArrayList<>();
+    private static String outputObjectMetricsLatency;
+    private static SimpleDateFormat latencyDateTimeFormatter;
```

Comment:
SimpleDateFormat is not thread-safe and should not be static...update to DateTimeFormatter.